forge-game/src/main/java/forge/game/trigger/Trigger.java

```diff
@@ -101,7 +101,7 @@ public Trigger(final Map<String, String> params, final Card host, final boolean
         this.mapParams.putAll(params);
         this.setHostCard(host);
 
-        String triggerZones = getParam("TriggerZones");
+        String triggerZones = getParamOrDefault("TriggerZones", "Battlefield");
```

Comment:
This would at least break all triggers with "When you cast this spell" (57 alone) and most likely a lot of other cards which use static helpers
I'm a bit concerned with such uncertain outcome tbh

Reply:
Fair enough; thanks